src/general/OptionsMenu.cs

```diff
@@ -394,6 +400,7 @@ public void ApplySettingsToControls(Settings settings)
             settings.CustomUsername :
             Environment.UserName;
         customUsername.Editable = settings.CustomUsernameEnabled;
+        goToScreenshotFolder.Pressed = settings.OpenScreenShotFolder;
```

Comment:
Shouldn't the button for the screenshot folder opening just be a normal button (that doesn't have pressed down state)?